src/components/tasks/single/singleTaskContent.tsx

```diff
@@ -86,9 +114,19 @@ const SingleTaskContent = ({ task }: SingleTaskProps) => {
                         {task.attachments.length > 0 && <div className="font-bold mt-1.5">Załączniki</div>}
                         {wrapAttachments(task.attachments)}
                     </div>
-                    {task.status !== TaskStatus.SOLVED && (
-                        <div className="flex-col w-full mt-5">
-                            <SolveForm onSubmit={onSubmit} />
+                    {status !== TaskStatus.SOLVED ? (
+                        <div className="w-full mt-5">
+                            <SolveForm loading={loading} backendErrors={errors} onSubmit={onSubmit} />
+                        </div>
+                    ) : (
+                        <div
+                            className="mt-5 relative px-4 py-3 leading-normal text-green-700 bg-green-100 rounded-lg"
```

Comment:
![image](https://user-images.githubusercontent.com/25308982/121784952-41e65580-cbb7-11eb-9e3e-7a6795c63205.png)
Całkiem fajnie to wygląda. Ten komunikat (zielone tło) mógłby być nawet rozszerzony na pełną szerokość modala zamiast ustawiać się na szerokość tekstu. 
Ale już teraz wygląda i działa ok.

Reply:
Zmienione

Reply:
Jest jeszcze lepiej :+1: